### PR TITLE
Add tests for AdmArea.get_facilities

### PR DESCRIPTION
**Pull request by operte**

### CODE REVIEW

File: tests/test_layers.py [hunk 1]
```diff
@@ -78,3 +78,32 @@ def test_get_country_data_level_1(self, mocker, capsys, multipolygon):
...
+    mocker.patch("gpbp.layers.AdmArea._get_country_data")
+    adm_area = AdmArea(country="Timor-Leste", level=0)
+    adm_area.geometry = multipolygon
+    adm_area.adm_name = "Timor-Leste"
+    return adm_area
+
+class TestAdmAreaGetFacilities:
+    def test_get_facilities_valid_method(self, mocker, adm_area):
+        mock_facilities_src = mocker.patch("gpbp.layers.FACILITIES_SRC", {"osm": mocker.Mock()})
+        tags = {"building":"hospital"}
```
Comment (dcossao) [inline tests/test_layers.py]: Simply a matter of taste: `{"building":"hospital”}` is used three times. If it becomes, I don’t know, `{“amenity":"hospital”}`, then we need to change it all over. Extract constant?

On the other hand, this way is perhaps more readable. Dealer’s choice

**Review by dcossao — COMMENTED** [review]
Tests look good, only one comment to discuss. 

Merge dev into this branch

File: tests/test_layers.py (hunk 1 above)
Comment (operte) [reply tests/test_layers.py]: Agreed. I don't know what the best practice would be here, a fixture, a static method, or a class attribute. I just went with fixture for no particular reason.

File: tests/test_layers.py [hunk 2]
```diff
@@ -89,22 +112,24 @@ def adm_area(mocker, multipolygon):
     adm_area.adm_name = "Timor-Leste"
     return adm_area
 
+@pytest.fixture
+def tags():
```
Comment (dcossao) [inline tests/test_layers.py]: I’d call it something like osm_hospital_tags because I think it would improve readability

Comment (operte) [reply tests/test_layers.py]: Good one, it was a bit too generic before.

**Review by dcossao — APPROVED** [review]

**Pull request merged**